common/on_action/birthday.txt

```diff
@@ -6,6 +6,9 @@ on_birthday = {
 		on_birthday_childhood
 		on_graceful_aging_birthday
 	}
+	events = {
+		WCNIC.0001
```

Comment:
For now, it's fine, but avoid adding 0 before the number.
There's no reason to do it.

Reply:
Done in 8b932ed0